llvm/test/MC/AArch64/SVE2/directive-arch-negative.s

```diff
@@ -6,10 +6,15 @@ tbx z0.b, z1.b, z2.b
 // CHECK: error: instruction requires: sve2 or sme
 // CHECK-NEXT: tbx z0.b, z1.b, z2.b
 
-.arch armv9-a+sve2-aes
```

Comment:
I have a feeling that the entire file is wrong. The pairs of lines `.arch <base>+<ext>` followed by `.arch <base>+<noext>`  should be replaced by a single line `.arch <base>+<ext>+<noext>`. Perhaps on a separate patch.

Reply:
Yep, I've been talking about this offline and plan to patch it up.

>Looks like we started adding these tests (https://reviews.llvm.org/D62602) because the behavior of .arch was broken (https://bugs.llvm.org/show_bug.cgi?id=32873) but seems to be fixed now.
Since [this patch](https://github.com/llvm/llvm-project/commit/a499d675ae163429adff0bc3dc4f8215c12441c7#diff-978708f6b40d44a13bced2d3050ddcd01f4a2423ca34ac0e26fb2ff2a83cc885R6943), as soon as we redeclare .arch X, the features available are reset to the defaults for X.